build.gradle

```diff
@@ -224,6 +225,12 @@ project('spring-rabbit') {
 		}
 
 		compile ("ch.qos.logback:logback-classic:$logbackVersion", optional)
+
+		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
+		compile ("org.apache.logging.log4j:log4j-jcl:$log4j2Version", optional)
```

Comment:
So you changed the test as well; that's different from saying "it's redundant".

"I changed the tests to use log4j2 directly" would have been more accurate :smile: